azure/notebooks/Train-RAPIDS.ipynb

```diff
@@ -217,11 +217,8 @@
    "source": [
     "import shutil\n",
     "\n",
-    "shutil.copy('train_rapids_RF.py', project_folder)\n",
-    "\n",
-    "# use rapids_csp scripts\n",
-    "# shutil.copy('train_azure.py', project_folder)\n",
-    "# shutil.copy('rapids_csp_azure.py', project_folder)"
+    "shutil.copy('train_rapids.py', project_folder)\n",
+    "shutil.copy('../code/rapids_csp_azure.py', project_folder)"
```

Comment:
Relative links can be a little unreliable in notebooks, if someone opens your notebook via jupyter lab, and they don't have the working directory you expect, this can break. It may be worthwhile to grab the real path to the file you're in, then reference other files based on that.